src/SFA.DAS.Authorization.WebApi/SFA.DAS.Authorization.WebApi.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;net462</TargetFrameworks>
```

Comment:
There's no separate WebApi in the .net core world (just Mvc.Core), so we should be able to remove it as a target framework.

Reply:
Unfortunately it looks like `Microsoft.AspNet.WebApi.WebHost` is .NET Framework?